dP256/src/test/kotlin/foundation/algorand/deterministicP256/DeterministicP256Test.kt

```diff
@@ -100,73 +105,414 @@ class DeterministicP256Test {
 
                         // Example values taken from: https://webauthn.guide/#registration
                         val origin = "https://webauthn.guide"
-                        val userId = "a2bd8bf7-2145-4a5a-910f-8fdc9ef421d3"
+                        val userHandle = "a2bd8bf7-2145-4a5a-910f-8fdc9ef421d3"
+
+                        val keyPair = D.genDomainSpecificKeypair(DerivedMainKey, origin, userHandle)
 
-                        val keyPair = D.genDomainSpecificKeypair(DerivedMainKey, origin, userId)
                         val keyPair0 =
                                 D.genDomainSpecificKeypair(
                                         DerivedMainKey,
                                         origin,
-                                        userId,
+                                        userHandle,
                                         counter = 0
                                 )
+
                         val keyPair1 =
                                 D.genDomainSpecificKeypair(
                                         DerivedMainKey,
                                         origin,
-                                        userId,
+                                        userHandle,
                                         counter = 1
                                 )
 
                         // Check generated public key against hardcoded value
                         assertEquals(
                                 keyPair.public.encoded.contentToString(),
-                                "[48, 89, 48, 19, 6, 7, 42, -122, 72, -50, 61, 2, 1, 6, 8, 42, -122, 72, -50, 61, 3, 1, 7, 3, 66, 0, 4, -51, -127, -4, -41, 35, 55, -122, -27, -104, 70, 74, -26, 65, 1, -45, -8, -22, 121, 109, -30, -70, 23, 113, -51, -9, 52, -77, 76, -102, -14, -80, 64, 89, -121, 14, -65, 0, 119, -104, 55, 22, 96, -13, -80, -23, -123, 41, 61, 91, -11, 47, 87, -49, -36, -10, 16, -112, -17, 96, 78, 102, -25, 71, 45]",
+                                "[48, 89, 48, 19, 6, 7, 42, -122, 72, -50, 61, 2, 1, 6, 8, 42, -122, 72, -50, 61, 3, 1, 7, 3, 66, 0, 4, 55, -123, -88, 32, 86, 59, 61, 35, 82, -35, 57, -71, 59, -12, 100, 95, -23, -122, 87, 60, -43, -59, -68, 118, -74, 82, -85, 97, -70, -60, -28, -73, -34, -86, 59, 65, -37, -108, -91, 120, 41, -95, -87, -1, -36, -68, -72, -78, -112, 95, -122, 97, 105, -112, -82, -104, -21, 19, 98, -49, 114, 59, -127, 76]",
```

Comment:
shouldn't that be the same as before?

Reply:
It's te same as the raw representation produced by Swift:

https://github.com/algorandfoundation/deterministicP256-swift/blob/d8ac95b2bcc3830556027a1d90a103d713681883/Tests/deterministicP256-swiftTests/deterministicP256_swiftTests.swift#L281

I believe there might have been a divergence in representation with the original implementation that was fixed when I switched it to BouncyCastle.